<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPanel</class>
 <widget class="QDialog" name="SettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;JetBrainsMonoNL NFM&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabPen">
    <attribute name="title">
     <string>画笔</string>
    </attribute>
    <widget class="QPushButton" name="penBtnPenColor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="penLabelPenColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="penLabelBrushColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>画刷颜色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="penBtnBrushColor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="penLabelPenWidth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔宽度</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="penSpinBoxPenWidth">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="penComboBoxPenMode">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelPenMode">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔模式</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
